src/vcf_file_generator.py

```diff
@@ -129,28 +131,28 @@ def _adjust_variant(self, variant):
             variant['POS'] = variant['start']
         elif so_term == 'MNV':
             variant['POS'] = variant['end']
+            if variant['POS'] == None:
+                return None
         else:
             logger.fatal('New SoTerm that We need to add logic for: %r', so_term)
             return None
         return variant
 
     def generate_files(self, skip_chromosomes=()):
         (assembly_chr_variants, assembly_species) = self._consume_data_source()
-
         for (assembly, chromo_variants) in assembly_chr_variants.items():
             logger.info('Generating VCF File for assembly %r', assembly)
-            filename = assembly + '-' + self.database_version  + '.vcf'
+            filename = assembly + '-' + self.database_version + '.vcf'
             filepath = os.path.join(self.generated_files_folder, filename)
+            print('INFO\t\t', assembly)
```

Comment:
same here: logger.info()

Reply:
Yep, same as above. Due to the error I had with mouse variants, it was better to have which assembly being output to debug.